electron/demokit/window.js

```diff
@@ -19,7 +19,7 @@ module.exports = async function window({ contentRect, template = DEFAULT_WINDOW_
         args: [{ contentRect: await calculateContentRect({ contentRect }), ...rest }, PRELOAD_PATH, uuid.v4(), template],
         script: function({ id, contentRect, contentURL, title }, PRELOAD_PATH, partition, template)
         {
-            const instantiation = document.getElementById("instantiation");
+            const instantiation = document.createElement("div");
```

Comment:
I can't for the life of me figure out why I used to do this the other way. Something that was necessary for the old electron?

Reply:
Was quite a challenge to discover this was the root cause of the issue I was having :)